src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import './Calculator.css';
+import Button from './Button';
+
+const Calculator = () => (
+  <div className="calculator">
+    <Button
+      text="0"
+      textAlign="right"
+      color="#858693"
+      textColor="#fff"
+      cursor="auto"
+    />
+    <div className="row">
+      <Button text="AC" />
+      <Button text="+/-" />
+      <Button text="%" />
+      <Button text="^" color="#f5913e" />
+    </div>
```

Comment:
- [ ] Your design looks nice :+1: , but please make sure that you follow the values of the calculator button in the design, As you can see below you used a power sign instead of the division sign :blush: 
  
  ### Your design
  
  ![image](https://github.com/andrianarivo/math-magicians/assets/92303891/bf282c7b-2d76-41fe-a516-92b9d9c8056e)
  
  ### Preferred design
  
  ![Screenshot from 2023-05-30 20-03-05](https://github.com/andrianarivo/math-magicians/assets/92303891/4ecb061a-1043-4644-96b3-542a9eacf309)